lib/rules/no-only-tests.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const utils = require('../utils');
+
+/**
+ * Checks if the given token is a comma token or not.
+ * From: https://github.com/eslint/eslint/blob/master/lib/rules/utils/ast-utils.js
+ * @param {Token} token The token to check.
+ * @returns {boolean} `true` if the token is a comma token.
+ */
+function isCommaToken (token) {
```

Comment:
@bmish 

Sorry, I'm late for this.

I added `eslint-utils` in #111, we should use those functions from it.

Reply:
Nice, didn't realize they came from there, fixed https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/pull/156